be/src/storage/rowset/indexed_column_reader.h

```diff
@@ -142,12 +140,12 @@ class IndexedColumnReader {
     Status read_page(RandomAccessFile* read_file, const PagePointer& pp, PageHandle* handle, Slice* body,
                      PageFooterPB* footer) const;
 
+    const IndexReadOptions& _opts;
+
     FileSystem* _fs;
     std::string _file_name;
     IndexedColumnMetaPB _meta;
 
-    bool _use_page_cache = true;
```

Comment:
Instead of deleting these two fields, how about add a new field `bool _skip_fill_local_cache` and remove your newly added field `const IndesReadOptions&`?

Reply:
ok